drivers/gpio/gpio_mchp_xec.c

```diff
@@ -170,8 +252,64 @@ static int gpio_xec_write(struct device *dev,
 
 }
 
-static int gpio_xec_read(struct device *dev,
-			 int access_op, u32_t pin, u32_t *value)
+static int gpio_xec_port_set_masked_raw(struct device *dev, u32_t mask,
+					u32_t value)
+{
+	const struct gpio_xec_config *config = dev->config->config_info;
+	u32_t port_n = config->port_num;
+
+	/* GPIO output registers are used for writing */
+	__IO u32_t *gpio_base = (__IO u32_t *)(GPIO_PAROUT_BASE
```

Comment:
Seems like a macro for this would be good.

Reply:
Created a macro for these.